website/archiver/listeners.py

```diff
@@ -27,6 +27,9 @@ def after_register(src, dst, user):
     archiver_utils.before_archive(dst, user)
     if dst.root != dst:  # if not top-level registration
         return
+    if dst.pending_embargo:
+        dst.archive_job.meta['embargo_urls'] = project_utils._get_embargo_urls(dst, user)
```

Comment:
Since `_get_embargo_urls` is used outside of the `utils` module, it probably shouldn't be marked as a private function with the `_` prefix.